wandb/sdk/artifacts/artifact_manifest_entry.py

```diff
@@ -150,7 +150,11 @@ def download(
 
         # Skip checking the cache (and possibly downloading) if the file already exists
         # and has the digest we're expecting.
-        if os.path.exists(dest_path) and self.digest == md5_file_b64(dest_path):
+        if (
+            os.path.exists(dest_path)
+            and not os.path.isdir(dest_path)
```

Comment:
we can't get the md5 hash for a directory, so we skip this check to avoid an error